### DISCUSSION TITLE
Does  keyvault-acmebot support 1 year certificates

### DISCUSSION THREAD

**Discussion by Patrickkk** [polymind-inc/acmebot#579] [Q&A] [answered]
Hello, awsome project you have here

Does it support 1 year certificates from for example https://zerossl.com/pricing/ we would like to be using the 1 year certificates while we are in a transitioning phase and some services still need manual updates for some parts of the system. I couldn't find any info in the documentation.

**Comment by shibayan** [comment]
I have only tried ZeroSSL with the Free plan, but the paid plan also supports ACME, so you should be able to get a 1-year certificate using EAB.

Perhaps a dedicated ACME Endpoint is provided for paid plan subscribers?

**Reply by Patrickkk** [reply, marked as answer]
Thank you for your response, we will just have to give it a try then :) 

Ill update this when i know if it works
